utils/frame/frame-utilities-cli/src/transfer.rs

```diff
@@ -0,0 +1,155 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implementation of the `transfer` cli subcommand for nodes that use the pallet-balances crate.
+
+use sc_cli::{
+    Error, SharedParams, pair_from_suri, with_crypto_scheme,
+	CryptoSchemeFlag, decode_hex, CliConfiguration, KeystoreParams,
+	GenericNumber,
+};
+use structopt::StructOpt;
+use std::{str::FromStr, fmt::Debug};
+use codec::{Encode, Decode};
+use sp_runtime::{MultiSigner, MultiSignature, AccountId32};
+use std::convert::TryFrom;
+use sp_core::{crypto::Ss58Codec, hexdisplay::HexDisplay};
+use frame_system::extras::{
+    AddressFor, IndexFor, AccountIdFor,
+    SignedExtensionProvider, CallFor,
+};
+use pallet_balances::Call as BalancesCall;
+use crate::utils::create_extrinsic_for;
+
+type Bytes = Vec<u8>;
```

Comment:
Why do we need this?

Reply:
structopt's custom parser only works with type-aliases due to some proc macro issues.